Apps/Common/src/Services/NonceService.cs

```diff
@@ -35,13 +36,13 @@ public NonceService()
             RNGCryptoServiceProvider rngProvider = new RNGCryptoServiceProvider();
             byte[] nonceBytes = new byte[32];
             rngProvider.GetBytes(nonceBytes);
-            this.currentNonce = Convert.ToBase64String(nonceBytes);
+            this.currentNonce = HttpUtility.HtmlEncode(Convert.ToBase64String(nonceBytes));
```

Comment:
What is the purpose of the HTML Encode?  Base64 is valid for HTML/URLs

Reply:
Transparently not. The front end (html) was getting the symbols jumbled up making the nonce not match the backend